frontend/src/app/confirm.service.spec.ts

```diff
@@ -80,3 +81,18 @@ describe('ConfirmService and its modal compoent', () => {
     expect(document.querySelector('.modal-content')).toBeFalsy();
   });
 });
+
+@Component({
+  template: ''
+})
+class TestComponent {
+}
+
+@NgModule({
+  imports: [RouterTestingModule, NgbModule.forRoot()],
+  declarations: [ConfirmModalContentComponent, TestComponent],
+  providers: [ConfirmService],
+  entryComponents: [ConfirmModalContentComponent]
+})
+class TestModule {
```

Comment:
I indeed used that when there are several configureTestingModule differing only by their providers. But it was also necessary here because I had to pass entryComponents, and that doesn't seem possible with configureTestingModule.